modules/extensions-mule-support/pom.xml

```diff
@@ -84,6 +81,15 @@
             <!-- the app classpath, this needs to go into the container classloader -->
             <scope>provided</scope>
         </dependency>
+        <dependency>
+            <!-- Needed for tests to be able to parse test config files -->
```

Comment:
config parsing tests should be moved to integration-tests

Reply:
moved here:
https://github.com/mulesoft/mule-integration-tests/pull/2695